projects/exui-common-lib/src/lib/components/search-location/search-location.component.spec.ts

```diff
@@ -450,4 +450,24 @@ describe('SearchLocationComponent', () => {
     component.getLocations('exampleString2');
     expect(locationServiceMock.getAllLocations).toHaveBeenCalledWith('api/locations/getLocations', 'IA,CIVIL', '', 'exampleString2', [{service: 'IA', locations: [{epimms_id: '12345'}]}, {service: 'CIVIL', locations: [{epimms_id: '32456'}]}]);
   });
+  // EUI-7909 - uncomment this code
```

Comment:
@johnbenjamin-hmcts - EUI-7909... has this been completed and if so, should this code be uncommented?

Reply:
@Daniel-Lam - I have updated the code.